app/controllers/toezicht/inzendingen.js

```diff
@@ -22,13 +23,17 @@ export default Controller.extend({
     this._super(...arguments);
     this.set('header', ENV['vo-webuniversum']['header']);
   },
-
-  filterChanged: observer('bestuurseenheidId', 'classificatieId', 'provincieId',
-                          'besluitTypeId', 'regulationTypeId', 'sessionDateFrom', 'sessionDateTo',
+  
+  filterChanged: observer('bestuurseenheidIds', 'classificatieIds', 'provincieIds',
+                          'besluitTypeIds', 'regulationTypeId', 'sessionDateFrom', 'sessionDateTo',
                           'sentDateFrom', 'sentDateTo', 'statusUri', function() {
     this.set('page', 0);
   }),
 
+  aRegulationIsSelected: computed('besluitTypes', function() {
+    return this.besluitTypes.some(type => type.isRegulation == true);
```

Comment:
`== true` isn't necessary. `type => type.isRegulation` is sufficient.

Reply:
I would change the name of the property to `regulationTypeIsSelected`, because it's about the type being selected, not a specific instance of a regulation.